_data/flux-commands.yaml

```diff
@@ -119,7 +127,42 @@ commands:
             command: flux alloc -N 2 --queue=foo
           - title: Create an allocation in the background with two nodes on queue 'foo'
             command: flux alloc --bg -N 2 --queue=foo
-            
+
+  - name: flux bulksubmit
+    groups:
+      - name: Format String Examples
+        items:
+          - title: Pipe inputs into flux bulksubmit
+            command: seq 1 4 | flux bulksubmit echo {}
+          - title: Map the same inputs into flux bulksubmit
```

Comment:
Maybe something like "Map the same inputs on the command line using GNU parallel-style `:::` separator" (give credit where credit is due)